vmm/src/vm.rs

```diff
@@ -483,8 +483,23 @@ impl VmmOps for VmOps {
     }
 }
 
-pub fn physical_bits(max_phys_bits: Option<u8>) -> u8 {
+pub fn physical_bits(max_phys_bits: Option<u8>, #[cfg(feature = "tdx")] tdx_enabled: bool) -> u8 {
```

Comment:
Perhaps a simpler solution would be to modify this to take in a reference to the config?

Reply:
Problem is, the `CpuManager` calls `physical_bits()` from `cpu.rs`, and I don't think it has access to the `Config`, on the `CpusConfig`.